### PR TITLE
Update install guide

### PR DESCRIPTION
**Pull request by i80and**
https://docs-mongodbcom-staging.corp.mongodb.com/meta/andrew/update-install-guide/tutorials/install.html

### CODE REVIEW

File: source/includes/steps-install.yaml [hunk 1]
```diff
@@ -1,67 +1,82 @@
...
      brew doctor
 
 ---
 title: Install Python 2 and Python 3
 ref: install-python
 content: |
-  
-  To run the build tools you need both python and python 2. Install them both by running:
+
+  To run the build tools you need both Python 3 and 2. Install them both by running:
```
Comment (schmalliso) [inline source/includes/steps-install.yaml]: We might emphasize that you can't/shouldn't use built-in Python, lest it ruin your life.

File: source/snooty.txt [hunk 2]
```diff
@@ -65,6 +65,12 @@ is loaded into Visual Studio Code with a ``snooty.toml`` file in the root.
    .. code-block:: sh
 
       git clone https://github.com/mongodb/docs-compass.git
+
+   Then open the ``docs-compass`` project either through the Visual Studio Code
+   UI or with the following command:
+
+   .. code-block:: sh
+
```
Comment (schmalliso) [inline source/snooty.txt]: Comment for line 169 (sorry, github doesn't let you comment on non-diff lines 😡 ): what's a workspace in this context? Does it mean I can only have docs-compass open, but not docs-compass AND another docs-whatever that's snooty-enabled?

File: source/includes/steps-install.yaml (hunk 1 above)
Comment (schmalliso) [inline source/includes/steps-install.yaml]: We might consider emphasizing that you need to use homebrew python2/3 *not* the built-in one, lest you ruin your life.

File: source/snooty.txt (hunk 2 above)
Comment (i80and) [reply source/snooty.txt]: Currently yeah, that's a limitation of snooty. I'll add a note to that effect!

File: source/includes/steps-install.yaml (hunk 1 above)
Comment (schmalliso) [reply source/includes/steps-install.yaml]: I'm not sure how I left this comment twice; I don't ACTUALLY feel that strongly about it 😁

Comment (i80and) [reply source/includes/steps-install.yaml]: Updated!

Comment (i80and) [reply source/includes/steps-install.yaml]: Updated!

**Pull request merged**